### PR TITLE
Add service account support to datacube-ows and explorer

### PR DESCRIPTION
**Pull request by woodcockr**
This PR adds Service Account support to datacube-ows chart. Necessary for IAM Roles for Service Account.

### CODE REVIEW

**Comment by woodcockr** [comment]
lint-docs is failing. I'm not familiar with lint-docs. So far as I can tell its just crashing. Not sure if that is the docs or the process itself?

**Comment by woodcockr** [comment]
@mmochan @NikitaGandhi @pindge Ping - PR needs a review please?

Also noticed that the explorer chart also has a service account branch that is sitting still. I'm currently running off a fork but would like to see SA support added to the charts. Any particular reason these can't progress?

**Comment by woodcockr** [comment]
I've amended this PR to include updating explorer chart as well - thought I saw someone else working on it on a somewhat stale branch but in looking again to merge them seems I was mistaken. Not to worry, this PR has them both now.

**Comment by NikitaGandhi** [comment]
We are bit thin at this movement so not sure anyone has a time to fix the pipeline checks 😞

**Comment by whatnick** [comment]
I have spoken to Rob and I will block time this weekend to look at this as
a public service.

File: stable/datacube-ows/values.yaml [hunk 1]
```diff
@@ -39,12 +50,14 @@ ows:
   livenessProbe: {}
   readinessProbe: {}
   startupProbe: {}
+  securityContext: {}
```
Comment (whatnick) [inline stable/datacube-ows/values.yaml:53]: @woodcockr - Could you please explain why we have security context twice and what the difference between the 2 of them is ?

File: stable/datacube-ows/values.yaml [hunk 2]
```diff
@@ -39,12 +50,14 @@ ows:
...
 owsConfig:
   image:
     registry: docker.io
     repository: geoscienceaustralia/dea-datakube-config
     tag: latest
     pullPolicy: Always
+  securityContext: {}
```
Comment (whatnick) [inline stable/datacube-ows/values.yaml:60]: Second security context here.

**Review by whatnick — CHANGES_REQUESTED** [review]
Dis-ambiguate and document security context.

File: stable/datacube-ows/values.yaml (hunk 1 above)
Comment (woodcockr) [reply stable/datacube-ows/values.yaml:53]: No idea, wasn't added by me and I didn't notice it ;-)


There are two Containers - which would be why there are two - if owsConfig uses an image for configuration.

@mmochan - What's the deal with the securityContext?

Comment (NikitaGandhi) [reply stable/datacube-ows/values.yaml:53]: securityContext is useful specify security settings for a Pod. Sometime pod running as non root user not able to read the AWS web identity token file. Then configuring `securityContext` can help. Not sure is that the intention here.

**Review by whatnick — APPROVED** [review]

**Pull request merged**